src/burnchains/db.rs

```diff
@@ -271,9 +272,25 @@ impl BurnchainDB {
         })
     }
 
+    pub fn get_burnchain_op(&self, txid: &Txid) -> Option<BlockstackOperationType> {
```

Comment:
This could be problematic if the burnchain itself forks, and the transaction gets mined on a nephew fork.  Is there code somewhere that either drops nephew forks, or only considers transactions if they're on the canonical burnchain fork?

Reply:
For how this is used, it won't matter -- this is only used to lookup an input's corresponding spent output. If the spending tx is in the current fork, then the corresponding txid would be as well (otherwise, the tx would never have been accepted by Bitcoin).

Reply:
This is true.